### ISSUE TITLE
A11y_OracleAssessmentForAzureDataStudio(ADS)_DatabaseMigrationAssessmentForOracle_DatabaseMigrationAssessmentForOracleDialog_Screen Reader: Screen reader is announcing the selected list item as unselected.

### ISSUE THREAD

**Issue by v-akr** [microsoft/azuredatastudio#19676]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win11;#DesktopApp;#WCAG1.3.1;#Benchmark;#FTP;#NVDA;#JAWS;

### Environment Details:
Application Name: Oracle Assessment for Azure Data Studio (ADS)
Operating System: Windows 11
Version :1.35.1
Screen reader: NVDA, JAWS

### Steps to Reproduce:
Download the azure data studio
Open the application and tab to the Connections icon in left navigation.
Tab to the new connections button and press enter. Connection dialog box will open.
Fill Correct details in connection details and tab to the connect button and press enter. New connection will be added.
Tab to the Database Migration Assessment for Oracle button and then press enter. "Database Migration Assessment for Oracle" page will be opened.
Tab to the "Run New Assessment" button at top the page and press enter. New Database Migration Assessment for Oracle dialog box will open.
Start NVDA then Tab till any combo-box and expand the combo-box and observe that Screen reader is announcing the selected list item as unselected

### Actual:
When pressing Esc key instead of dismissing the tooltip whole dialog is getting dismissed.

### Expected:
When pressing Esc key tooltip should be dismissed not the whole dialog.

### User Impact:
Keyboard users will face difficulty because by pressing esc key tooltip is not getting dismissed as tooltip is hiding content.
